docs/pages/installation/quick-start.md

```diff
@@ -83,3 +83,10 @@ You may now proceed to install and work with any [service mesh]({{ site.baseurl
  Your browser does not support the video tag
 </video>
 
+
+## Additional Guides
```

Comment:
It would be good to pre-pend the site base URL variable before each of these hyperlinks

Reply:
Sure ! I did in both the ways . Sorry , I was not able to push the new changes I will create a new PR.